### PR TITLE
typo missing ')'

### PR DESCRIPTION
**Pull request by cloudbank**
it is missing the ) at end of android_ndk_repository(

### CODE REVIEW

**Comment by cloudbank** [comment]
I signed it

File: WORKSPACE [hunk 1]
```diff
@@ -37,7 +37,7 @@ load("//tensorflow:workspace.bzl", "tf_workspace")
 #    # API level that it supports without notice.
 #    # Note that the NDK version is not the API level.
 #    api_level=14)
```
Comment (Corea) [inline WORKSPACE]: Here's closing `)`!

**Review by jhseu — APPROVED** [review]

File: WORKSPACE (hunk 1 above)
Comment (cloudbank) [inline WORKSPACE]: #    api_level=14)
this line was missing in my file for some reason
and it is good that you noticed.....
because my build needs it

**Comment by jhseu** [comment]
Jenkins, test this please

**Comment by cloudbank** [comment]
@jhseu 
Should I close this?  My copy of the file was missing the line 
    api_level=14)
for some reason and don't want to add an unnecessary comment to the file

**Comment by jhseu** [comment]
Ah, ok, closing

**Pull request closed without merging**